pinot-common/pom.xml

```diff
@@ -301,6 +301,11 @@
       <groupId>org.apache.yetus</groupId>
       <artifactId>audience-annotations</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.mindrot</groupId>
+      <artifactId>jbcrypt</artifactId>
+      <version>0.4</version>
+    </dependency>
```

Comment:
@Jackie-Jiang do we have bcrypt available somewhere in the existing dependencies?

Reply:
@mayankshriv is bcrypt available already? otherwise is the license compatible? https://github.com/jeremyh/jBCrypt/blob/master/LICENSE

Reply:
Thanks @apucher. If this is a new dependency being added, licensing might not be compatible with Apache License. Are there alternatives to be used here?

Reply:
The license for `jBCrypt`looks to be not the standard license like BSD 2,3 clause or MIT. Is it possible to use this instead? Spring is based on apache license 
https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/crypto/bcrypt/BCrypt.html